generate.py

```diff
@@ -128,18 +140,22 @@ def fix(obj):
 with fixed_schema_path.joinpath('top.json').open('w') as f:
     json.dump(top, f, sort_keys=True, indent=4)
 
-# Convert the JSON schemas into Python objects using statham. These
-# form our return types.
-schema = materialize(RefDict(str(fixed_schema_path.joinpath('top.json'))), context_labeller=title_labeller())
+# Convert the JSON schemas into Python objects using statham.
+# These form our return types.
+curr_dir = Path.cwd()
+os.chdir(fixed_schema_path)
+schema = materialize(RefDict('top.json'), context_labeller=title_labeller())
+os.chdir(curr_dir)
```

Comment:
What's up with the `chdir`? `RefDict(fixed_schema_path.joinpath('top.json')` worked OK for me?

Reply:
It didn't for me.... and this way we could run the script from any folder actually

Reply:
That's where I got confused. There are other sections of the code that require running from the same directory as `generate.py`. Picking up the template files and writing the `.py` files below, for example. I'll run some tests later tonight to work out what is going on.

Reply:
I just ran a test without the `chdir` and using the full relative path to `top.json`. Works fine, both with your code and mine. Tested both your code and mine from a different directory, and both fail.